src/Migrate/custom/New-AzMigrateServerReplication.ps1

```diff
@@ -119,6 +127,36 @@ function New-AzMigrateServerReplication {
         # Specifies the Availability Zone to be used for VM creation.
         ${TargetAvailabilityZone},
 
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20210210.IVMwareCbtEnableMigrationInputTargetVmtags]
+        # Specifies the tag to be used for VM creation.
+        ${VMTag},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20210210.IVMwareCbtEnableMigrationInputTargetNicTags]
+        # Specifies the tag to be used for Nic creation.
+        ${NicTag},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20210210.IVMwareCbtEnableMigrationInputTargetDiskTags]
+        # Specifies the tag to be used for Disk creation.
+        ${DiskTag},
```

Comment:
Keep it ~ to service, SeedDiskTags and TargetDiskTags

Reply:
Kept same as suggested in PM doc.